content/content-tool/tool/src/webapp/vm/content/sakai_filepicker_attach.vm

```diff
@@ -100,11 +100,12 @@ includeLatestJQuery('sakai_filepicker_attach.vm');
 	</script>
 
 	<script>
-		document.addEventListener("DOMContentLoaded", function(event){ 
-			$('#resourceTabs a').click(function (e) {
+		document.addEventListener('DOMContentLoaded', () => {
+  			const tabLinks = document.querySelectorAll('#resourceTabs .nav-link');
+			tabLinks.forEach(link => link.addEventListener('click', e => {
```

Comment:
You don't need the tabLinks variable here. Just append .forEach on to the selector.

Reply:
It's amazing, how you think :)

Reply:
Thanks for the no-variable solution.